src/v8/v8_string.rs

```diff
@@ -42,6 +57,48 @@ impl<'isolate_scope, 'isolate> V8LocalString<'isolate_scope, 'isolate> {
     }
 }
 
+impl<'isolate_scope, 'isolate> std::fmt::Debug for V8LocalString<'isolate_scope, 'isolate> {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        let string_value = String::from(self.to_owned());
+        #[allow(dead_code)]
+        #[derive(Debug)]
+        struct V8StringDebugPrinter {
+            address: *mut v8_local_string,
+            value: String,
+        }
+        let inner_string = V8StringDebugPrinter {
+            address: self.inner_string,
+            value: string_value,
+        };
+        f.debug_struct("V8LocalString")
+            .field("inner_string", &inner_string)
+            .field("isolate_scope", self.isolate_scope)
+            .finish()
+    }
+}
+
+impl<'isolate_scope, 'isolate> From<V8LocalString<'isolate_scope, 'isolate>> for String {
+    fn from(value: V8LocalString<'isolate_scope, 'isolate>) -> Self {
+        String::from(&value)
+    }
+}
+
+impl<'isolate_scope, 'isolate> From<&V8LocalString<'isolate_scope, 'isolate>> for String {
+    fn from(value: &V8LocalString<'isolate_scope, 'isolate>) -> Self {
+        value
+            .to_value()
+            .to_utf8()
+            .map(|utf| utf.as_str().to_owned())
+            .expect("The V8LocalString isn't a valid UTF8 string.")
+    }
+}
+
+impl<'isolate_scope, 'isolate> Clone for V8LocalString<'isolate_scope, 'isolate> {
+    fn clone(&self) -> Self {
+        Self::new(self.isolate_scope, &String::from(self))
```

Comment:
Maybe this can be done more efficient? Instead of convert to rust string and then back to `V8LocalString`?